src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const ulList = [...document.querySelector('ul').children];
+const newList = document.querySelector('ul');
+
+function formatSalary(employee) {
```

Comment:
it's better if your function accept salary as an argument. It will be more clear